common/src/main/java/com/microsoft/identity/common/internal/cache/CacheRecord.java

```diff
@@ -48,7 +50,7 @@ public CacheRecordBuilder mAccount(final @NonNull AccountRecord account) {
             if (account == null) {
                 throw new NullPointerException("The account record for a CacheRecord may not be null");
```

Comment:
I think you could write a lombok non-null annotation on the variable, and then the builder will respect it.